### ISSUE TITLE
light version

### ISSUE THREAD

**Issue by kobe233333** [vztu/VIDEVAL_release#10]

**Comment by kobe233333** [comment]
When i try to use light-version to get features of my own 4k dataset, i got error when i set the max_reso = 1080;
frs_per_blk = 6;
the error is:
对于此运算，数组的大小不兼容。

出错 calc_VIDEVAL_feats_light>compute_tlvqm_lcf_features (第 940 行)
        static_area_flicker_bw = sum(non_mot_area(:) .* ...

出错 calc_VIDEVAL_feats_light (第 106 行)
        tlvqm_lcf_feats = compute_tlvqm_lcf_features(this_YUV_frame./255, ...

出错 demo_compute_VIDEVAL_light_feats (第 81 行)
        feats_mat(i,:) = calc_VIDEVAL_feats_light(yuv_name, ...

**Comment by kobe233333** [comment]
when i set the max_reso = 2160; frs_per_blk = 6; the process is good;
but if i set max_reso smaller than 2160, its not worked.

**Comment by vztu** [comment]
Hi, this error occurs in `compute_tlvqm_lcf_features` function that I borrowed from TLVQM. I am not very sure about the details in this function, so you'd have to debug yourself to see which step is wrong. I recommend you to run it line-by-line to check the matrix shapes. 

I recommend you to use the full VIDEVAL model if computation is not a problem. I added the light model in a hurry so it's possible that there might be some bug.
